README.md

```diff
@@ -259,6 +259,7 @@ The Anki version (a clone of the +200 flashcards from this repo) is available vi
 * [TCP connection backlog](network.md#tcp-connection-backlog)
 * [TCP flow control](network.md#tcp-flow-control)
 * [TCP handshake](network.md#tcp-handshake)
+* [TCP Connection Termination](network.md#tcp-connection-termination)
```

Comment:
```suggestion
* [TCP connection termination](network.md#tcp-connection-termination)
```

Reply:
Why has it been removed?

Reply:
Sorry for misunderstanding your previous review comment, I will fix it.